docs/Evaluation.md

```diff
@@ -40,6 +107,8 @@ results["box_precision"]
 0.781
 ```
 
+### Worked example of calculating IoU and recall/precision values
+
 To convert overlap among predicted and ground truth bounding boxes into measures of accuracy and precision, the most common approach is to compare the overlap using the intersection-over-union metric (IoU).
```

Comment:
IoU is the ratio between the area of overlap between the predicted polygon box and the ground truth polygon box divided by the area of the combined bounding box region.